src/cluster/src/start/k8.rs

```diff
@@ -1032,6 +1049,34 @@ impl ClusterInstaller {
         }
     }
 
+    /// Wait until the platform version of the cluster matches the chart version here
+    #[instrument(skip(self))]
+    async fn wait_for_fluvio_version(&self) -> Result<(), K8InstallError> {
```

Comment:
use `FluvioConfig` rather than rely on profile since it might not be available (there is option to skip profile save).  See: `create_managed_spu_group`